src/app/story/directives/status.component.ts

```diff
@@ -1,64 +1,43 @@
 import { Component, Input, DoCheck } from '@angular/core';
-import { Router } from '@angular/router';
-import { Angulartics2 } from 'angulartics2';
-import { ModalService, ToastrService } from 'ngx-prx-styleguide';
+import { ModalService } from 'ngx-prx-styleguide';
 import { StoryModel } from '../../shared';
 
+export type StoryStatus = 'draft' | 'scheduled' | 'published';
 @Component({
   selector: 'publish-story-status',
   styleUrls: ['status.component.css'],
   template: `
-    <h1>Publish</h1>
+    <h2 [class]="currentStatus">{{currentStatus}}</h2>
     <dl>
-
       <dt>Status</dt>
```

Comment:
I'm a little bothered by the semantics of these inputs not having labels in this `dl` `dt`/`dd`. Maybe Peter just kept it because it's what was there?

Reply:
was just an accessibility nitpick, but labels also dont seem as useful in this box.

Reply:
bolded but otherwise left alone